### PR TITLE
Fixed for issue #57

### PR DESCRIPTION
**Pull request by atapatel**
Try to fix issue #57 .

### CODE REVIEW

**Review by lamasfoker — CHANGES_REQUESTED** [review]
This have to be the correct way
`{{ money.convertAndFormat(result.priceByChannelAndCurrency(channel.code, currencyCode).value) }}`

**Review by atapatel — COMMENTED** [review]
Looks Good

File: src/Resources/views/Common/_box.html.twig [hunk 1]
```diff
@@ -21,7 +21,7 @@
         <a href="#" class="header sylius-product-name">{{ result.attribute('name').value | join }}</a>
 
         {% if result.price is not empty %}
-            <div class="sylius-product-price">{{ money.format(result.priceByChannelAndCurrency(channel.code, currencyCode).value, currencyCode) }}</div>
+            <div class="sylius-product-price">{{ money.convertAndFormat(result.priceByChannelAndCurrency(channel.code, currencyCode).value, currencyCode) }}</div>
```
Comment (jacquesbh) [inline src/Resources/views/Common/_box.html.twig:24]: Could you update the call to fit the method signature please?

https://github.com/Sylius/Sylius/blob/981021539e4e66a97f8b1d1d8ebed8dead1b3d96/src/Sylius/Bundle/ShopBundle/Resources/views/Common/Macro/money.html.twig#L5-L9

**Review by jacquesbh — CHANGES_REQUESTED** [review]

File: src/Resources/views/Common/_box.html.twig (hunk 1 above)
Comment (atapatel) [reply src/Resources/views/Common/_box.html.twig:24]: Resolved.

Comment (jacquesbh) [reply src/Resources/views/Common/_box.html.twig:24]: Sorry but no, it's not. Did you forget to push?

**Comment by lamasfoker** [comment]
Close this PR and merge #71 instead

**Comment by jacquesbh** [comment]
Closed by #71.

**Pull request closed without merging**